### ISSUE TITLE
No such file or directory: '../models/darknet/yolov2/fp32/yolo.xdnn.nobn.608.cfg'

### ISSUE THREAD

**Issue by CainXIII** [Xilinx/ml-suite#85]
# Run the darknet2caffe python script which will convert a yolov2 darknet model to a caffe model
# This script additionally will remove batchnorm layers, by merging the parameters
# This step is intentionally performed upstream of the compiler due to Darknet's non-conforming implementation of 
#   batch normalization

config["prototxt"] = "./work/yolov2.prototxt" 
config["caffemodel"] = "./work/yolov2.caffemodel"

darknet2caffe("../models/darknet/yolov2/fp32/yolo.xdnn.nobn.608.cfg",
              "../models/darknet/yolov2/fp32/yolo.xdnn.nobn.weights",
              config["prototxt"],
              config["caffemodel"])

i cann't find the path to yolo.xdnn.nobn.608.cfg and neither yolo.xdnn.nobn.weights
can someone help me?
